test/test_scratchpad.py

```diff
@@ -65,6 +70,68 @@ def is_killed(manager, name):
     raise ValueError('not yet killed')
 
 
+@scratchpad_config
+def test_sratchpad_with_matcher(manager):
+    # adjust command for current display
+    manager.c.group["SCRATCHPAD"].dropdown_reconfigure('dd-e', command='xterm -T dd-e -class dd-e -display %s sh' % manager.display)
```

Comment:
Is manually specifying the display necessary? I think spawning commands in the tests should have the display present in their environment. If not then making that the case might make more sense than having to do this manul specification.

Reply:
Without specifying the display, test fails. Other tests in the samefile too specifies display.